packages/node/src/login/oidc/refresh/TokenRefresher.ts

```diff
@@ -107,30 +122,25 @@ export default class TokenRefresher implements ITokenRefresher {
       );
     }
 
-    const tokenSet = await client.refresh(refreshToken, {
-      // openid-client does not support yet jose@3.x, and expects
-      // type definitions that are no longer present. However, the JWK
-      // type that we pass here is compatible with the API, hence the `any`
-      // assertion.
-      DPoP: dpopKey ? (dpopKey.privateKey as KeyObject) : undefined,
-    });
-
-    if (tokenSet.access_token === undefined) {
-      // The error message is left minimal on purpose not to leak the tokens.
-      throw new Error(
-        `The Identity Provider [${issuer.metadata.issuer}] did not return an access token on refresh.`
-      );
-    }
+    const tokenSet = tokenSetToTokenEndpointResponse(
+      await client.refresh(refreshToken, {
+        // openid-client does not support yet jose@3.x, and expects
+        // type definitions that are no longer present. However, the JWK
+        // type that we pass here is compatible with the API, hence the `any`
+        // assertion.
```

Comment:
I don't see `any` anymore :)

```suggestion
        // type that we pass here is compatible with the API, hence the type
        // assertion.
```